haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku. \
+Intended to be a full-featured organizer. "
+HOMEPAGE="https://sourceforge.net/projects/eventual/"
+COPYRIGHT="2016(?) Alex Hitech" #Help on this one
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="846290b16e76a8a4ffbd955057306a18ac96d5e3"
+SOURCE_URI="https://github.com/HaikuArchives/Eventual/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="e0b9844cebb6ed2cf42f93e592fe2e4ce0b801457b31cbac7ff7f5b5d493f76e"
+SOURCE_FILENAME="Eventual-$srcGitRev.tar.gz"
+SOURCE_DIR="Eventual-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86_64"
+
+PROVIDES="
+	eventual = $portVersion
+	app:Eventual = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:sh
+	"
+
+BUILD()
+{
+	sh makelib.sh
+	cd src/Libraries
+	for LIBDIR in */ ; do
+		LIBDIR=${LIBDIR%?}
+		NAME=lib$LIBDIR.so
+		LIB=$LIBDIR/objects.x86-cc2-release/$NAME
+		cp $LIB ../EventEditor/$NAME
+		cp $LIB ../EventServer/$NAME
+		cp $LIB ../PreferencesPreflet/$NAME
+		cp $LIB /boot/home/config/non-packaged/lib/$NAME #Error here
```

Comment:
package libs should be put in $appsDir/Eventual/lib

Reply:
Okay, @diversys.
I'll make the commit, but I can't test it :(